bridge/Mixer.cpp

```diff
@@ -788,6 +790,10 @@ bool Mixer::getEndpointInfo(const std::string& endpointId,
         if (audio->second)
         {
             foundAudio = true;
+            if (audio->second->remoteSsrc.isSet())
+            {
+                ssrc.set(audio->second->remoteSsrc.get());
+            }
```

Comment:
[Minor] If `audio->second->remoteSsrc.isSet()`, `ssrc` should be empty also, then let's simplify the code:
```suggestion
            ssrc = audio->second->remoteSsrc;
```

Reply:
Agree